_data/sites.json

```diff
@@ -19578,7 +19578,7 @@
     {
         "name": "Webex - Free Account",
         "url": "https://help.webex.com/en-us/5m4i4y/Delete-Your-Free-Webex-Account#id_111643",
-        "difficulty": "easy",
+        "difficulty": "medium",
         "notes": "Just follow the settings to find the delete account button and click it",
```

Comment:
Is this change supposed to be here?

Reply:
Yes, it was :)

Reply:
But why? I'd expect a longer notes section if the difficulty turned to medium

Reply:
Oooh, I now see I've changes the difficulty on the wrong site. It was supposed to be for webhosting.dk. Sorry.